# Prefer lodash-es to lodash for better tree-shaking support (`ben/no-lodash`)

âš ï¸ This rule _warns_ in the âœ… `recommended` config.

ğŸ”§ This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

## Rule Details

ì ì§„ì ìœ¼ë¡œ `lodash-es`ë¡œ ë§ˆì´ê·¸ë ˆì´ì…˜í•˜ì—¬ `lodash` íŒ¨í‚¤ì§€ë¥¼ ì œê±°í•˜ê¸° ìœ„í•¨ì…ë‹ˆë‹¤.

Examples of **incorrect** code for this rule:

```js
import _ from 'lodash';
```

Examples of **correct** code for this rule:

```js
import { isArray } from 'lodash-es';
```

## When Not To Use It

`lodash`ì—ì„œ `lodash-es`ë¡œ ë§ˆì´ê·¸ë ˆì´ì…˜ì´ ì™„ë£Œëœ í›„, `lodash` íŒ¨í‚¤ì§€ë¥¼ ì‚¬ìš©í•˜ì§€ ì•Šê²Œ ë˜ë©´ ì´ ê·œì¹™ì„ ì‚¬ìš©í•˜ì§€ ì•ŠëŠ” ê²ƒì´ ì¢‹ìŠµë‹ˆë‹¤.
