# Prefer date-fns to moment for better tree-shaking support (`ben/no-moment`)

âš ï¸ This rule _warns_ in the âœ… `recommended` config.

ğŸ”§ This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

Please describe the origin of the rule here.

## Rule Details

ì ì§„ì ìœ¼ë¡œ `date-fns`ë¡œ ë§ˆì´ê·¸ë ˆì´ì…˜í•˜ì—¬ `moment` íŒ¨í‚¤ì§€ë¥¼ ì œê±°í•˜ê¸° ìœ„í•¨ì…ë‹ˆë‹¤.

Examples of **incorrect** code for this rule:

```js
import moment from 'moment';
```

Examples of **correct** code for this rule:

```js
import { format } from 'date-fns';
```

## When Not To Use It

`moment`ì—ì„œ `date-fns`ë¡œ ë§ˆì´ê·¸ë ˆì´ì…˜ì´ ì™„ë£Œëœ í›„, `moment` íŒ¨í‚¤ì§€ë¥¼ ì‚¬ìš©í•˜ì§€ ì•Šê²Œ ë˜ë©´ ì´ ê·œì¹™ì„ ë¹„í™œì„±í™”í•  ìˆ˜ ìˆìŠµë‹ˆë‹¤.
